sample/src/main/java/kin/sdk/core/sample/ExportKeystoreActivity.java

```diff
@@ -43,7 +45,7 @@ private void initWidgets() {
 
         findViewById(R.id.copy_btn).setOnClickListener(view -> {
             selectAll();
-            ViewUtils.copyToClipboard(this, outputTextView.getText());
+            BaseDialog.copyToClipboard(this, outputTextView.getText());
```

Comment:
copyToClipboard has nothing to do with the BaseDialog.
I think we should keep ViewUtils just for that method